### PR TITLE
Delete w/ whitespace fix

### PR DESCRIPTION
**Pull request by ChipWolf**
Probably errors and such, but this is what needs to be done @GrahamCampbell

### CODE REVIEW

**Comment by ChipWolf** [comment]
Finally. StyleCI likes me :heart_eyes_cat:

File: src/Http/Controllers/RemoveController.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    protected function fire()
+    {
+        $image = $this->container->get('factory')->remove($this->args['id']);
+
+        if (!$image) {
+            return $this->success(['message' => 'Image removed'], 410);
+        }
+
+        if ($image) {
+            return $this->error(['message' => 'Image not removed. An error occured.'], 500);
```
Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:35]: Error 404 would make more sense probably. If we couldn't delete the image, it probably didn't exist.

File: src/Http/Controllers/RemoveController.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        return $this->error(['message' => 'Not Found'], 404);
```
Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: This lines is never executed. One of the earlier lines is always hit first.

File: src/Http/Controllers/RemoveController.php (hunk 1 above)
Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:35]: That's 'if image still exists'

File: src/Http/Controllers/RemoveController.php (hunk 2 above)
Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:38]: There needs to be a way to differentiate between a 404 and a successful deletion

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: Yeh, but this line is never executed. Image is never neither truthy or falsely, so this line is never hit.

Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:38]: I'm aware. So how can we differentiate between a 404 and a successful deletion

Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:38]: We have to also prevent old IDs from being used

Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:38]: So, I guess we could add another check to if(!$image) to see if it used to exist

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: > There needs to be a way to differentiate between a 404 and a successful deletion

Yeh. I should update the remove method to throw an exception.

> We have to also prevent old IDs from being used

We actually already do. Under the hood we're using auto-incrementing integer ids.

Comment (MichaelBanks) [inline src/Http/Controllers/RemoveController.php]: How many different combinations do we have in our valuespace. I'm assuming 10*26*26*4 (numbers * upper case * lower case * length)

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php]: > How many different combinations do we have in our valuespace.

I don't know what you mean? The "encoded" id the user is given is different.

Comment (MichaelBanks) [inline src/Http/Controllers/RemoveController.php]: But, there's only a limited amount of them

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: There's way more than that. We're using UNSIGNED BIGINT. We've got 18446744073709551615. :)

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: http://dev.mysql.com/doc/refman/5.6/en/integer-types.html

Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:38]: so why do Imgur bother with 6 characters?!

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: Oh. I see what you mean. Our system will just use more chars when we run out of 4 char combinations. :)

Comment (ChipWolf) [inline src/Http/Controllers/RemoveController.php:38]: According to my calculations we have 13,388,280 possible combinations in 4 chars.

Comment (GrahamCampbell) [inline src/Http/Controllers/RemoveController.php:38]: As, I said we already just use more characters automatically.

**Comment by GrahamCampbell** [comment]
@MichaelBanks Could you please send this to the 1.1 branch.

**Pull request closed without merging**